Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_038.PostgreSqlCredentials.ps1

```diff
@@ -7,10 +7,10 @@ Server=some-database-name.postgres.database.azure.com;Database=catalog_db;Port=5
 # Currently caught
 <add name="BlogDbContext" connectionString="host=my_host;Port=5555;Database=my_db;Username=my_user;Password=pass_1111"> # host not capitalized, optional properties in the middle
 <add name="BlogDbContext" connectionString="host=my_host;Database=my_db;Username=my_user;Password=pass_2222;Port=5555"> # optional properties in the middle and at the end
-<add name="BlogDbContext" connectionString="Host=my_host;Port=5555;UsErname=my_user;Password=pass_3333"> # weird capitalization on UsErname
-<add name="BlogDbContext" connectionString="Password=pass_4444;Host=my_host;Port=5555;Username=my_user;Pooling=true"> # weird order, unnecessary property at the end
-<add name="BlogDbContext" connectionString="Port=5555;Password=pass_5555;Connection Pruning Interval=doodle;Host=my_host;Username=my_user;Pooling=true"> # begin with optional property
-<add name="BlogDbContext" connectionString="Port=5555;Password=pass_6666;Connection Pruning Interval=doodle;Host=my_host;Username=my_user;Pooling=true;Port=5555"> # redundant optional property at beginning and end
+<add name="BlogDbContext" connectionString="Host=my_host;dbname=my_db;Port=5555;UsErname=my_user;Password=pass_3333"> # weird capitalization on UsErname
```

Comment:
>dbname [](http://example.com/codeflow?start=57&length=6)

Check it out! Previously, these patterns were incomplete, they did not have the database property. But because all properties were optional, we called the static validator! When I fixed that I had to introduce this additional data to ensure the next phase is invoked.